scripts/authoring_env.sh

```diff
@@ -14,46 +14,54 @@ help () {
     set-variant    Configure content variant used for preview
     reset-variant  Reset variant used for preview to default
     create-variant Create a variant HTML file for page
+    start-editing  Initialize environment for new round of edits
     "
 }
 
+start_env () {
+    docker compose up --build -d
+    docker compose exec moodle ./wait-for-it.sh postgres:5432 -- php admin/cli/install_database.php --agree-license --fullname="Local Dev" --shortname="Local Dev" --summary="Local Dev" --adminpass="admin" --adminemail="admin@acmeinc.com"
+    docker compose exec postgres psql -U moodle -d moodle -c "update mdl_config set value='1' where name='forcelogin'"
```

Comment:
Does this automate the process that we went through yesterday with viviane?

Reply:
I don't think so. Most of what we went through with Vivianne was her Docker setup (and then the Windows line encoding issue which prevented the scripts from running properly). She did eventually get `./scripts/authoring_env.sh` to run IIRC, but for both her and Susan they still didn't have a course in the instance. Up until now a user had to manually do that via the Site Administration menu after running a different script to generate the `mbz` files. The new command will automate all of that so at the end you are in a feature branch with a brand new Moodle that has the latest `mbz` loaded for you.

Reply:
Is it possible to automate the Docker setup line encoding fix when it happens on windows? Or do we not know enough about that issue to do such a thing?

Reply:
Yeah, no, I wouldn't try to do that here, and I don't think anything we did fixed it anyways. Vivian said she just had to reboot and then it prompted her to do something (I didn't catch what) and things worked. We can perhaps ask her the details on Monday and also check with Peter since I think he was on a PC too. To be clear, what Vivian hit wasn't a Docker issue, it was a Windows vs the rest of the world issue. [This article](https://www.aleksandrhovhannisyan.com/blog/crlf-vs-lf-normalizing-line-endings-in-git/) has a pretty good synopsis of the history of CRLF vs LF.